packages/eas-cli/src/build/createContext.ts

```diff
@@ -33,6 +35,7 @@ export async function createBuildContextAsync<T extends Platform>({
   nonInteractive: boolean;
   platform: T;
   projectDir: string;
+  resourceClass?: BuildResourceClass;
```

Comment:
This shouldn't be optional here. We already know the exact value after converting user input to graphql enum.